LongWarOfTheChosen/Src/WotCWeaponsAndArmor/Classes/X2Item_Weapons.uc

```diff
@@ -0,0 +1,923 @@
+// This is an Unreal Script
+class X2Item_Weapons extends X2Item config(GameData_WeaponData);

```

Comment:
To be consistent with the LW2 way of doing things, these weapons should probably be added to the [`ItemTable` array](https://github.com/long-war-2/lwotc/blob/master/LongWarOfTheChosen/Config/XComLW_Overhaul.ini#L448) in `XComLW_Overhaul.ini`.

Reply:
Ahh ok so if I add them to that file I can edit the costs there and not have to worry about setting the costs at template creation then?

Reply:
Correct. These values are applies to existing templates in the `LWTemplateMods` class.

Reply:
These are now added to the itemtable array in LW_Overhaul.ini